src/controllers/users.controller.ts

```diff
@@ -60,6 +60,18 @@ class UsersController {
       next(error);
     }
   };
+
+  public manageGroups = async (req: Request, res: Response, next: NextFunction) => {
```

Comment:
If you take a look at the rest of the file, we have `Promise<void>` in the return type of the functions. Although, I think that doesn't add much value, certainly, is unconventional.